tools/csv-merger/csv-merger.go

```diff
@@ -516,30 +509,24 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 	templateStrategySpec := &ocsCSV.Spec.InstallStrategy.StrategySpec
 
 	// Merge CSVs into Unified CSV
-	for _, csv := range mergeCsvs {
-		if csv == noobaaCSV {
-			continue
-		} else {
-			strategySpec := csv.Spec.InstallStrategy.StrategySpec
-
-			deploymentspecs := strategySpec.DeploymentSpecs
-			clusterPermissions := strategySpec.ClusterPermissions
-			permissions := strategySpec.Permissions
-
-			templateStrategySpec.DeploymentSpecs = append(templateStrategySpec.DeploymentSpecs, deploymentspecs...)
-			templateStrategySpec.ClusterPermissions = append(templateStrategySpec.ClusterPermissions, clusterPermissions...)
-			templateStrategySpec.Permissions = append(templateStrategySpec.Permissions, permissions...)
-
-			for _, definition := range csv.Spec.CustomResourceDefinitions.Owned {
-				// do not add vr and vrc to csv, this will be owned by csi-addons now.
-				if !(definition.Name == "volumereplications.replication.storage.openshift.io" ||
-					definition.Name == "volumereplicationclasses.replication.storage.openshift.io") {
-					ocsCSV.Spec.CustomResourceDefinitions.Owned = append(ocsCSV.Spec.CustomResourceDefinitions.Owned, definition)
-				}
-			}
+	strategySpec := rookCSV.Spec.InstallStrategy.StrategySpec
```

Comment:
Let's have the for loop, We will require it when shipping Prometheus with the ocs.

Reply:
Updated